steps/rhelah.py

```diff
@@ -484,25 +484,33 @@ def step_impl(context, name=None):
 
 
 @when(u'atomic mount image "{name}" to a specified "{mountpoint}"')
-def step_impl(context, name, mountpoint):
+@when(u'atomic mount image "{name}" with "{option}" to a specified "{mountpoint}"')
+def step_impl(context, name, mountpoint, ignore_rc=False, option=""):
     '''mount image to a specified directory'''
+    context.mount_option = option
```

Comment:
`contex.mount_option` appears to be unused elsewhere...is this assignment necessary?